src/components/endowed-positions/EndowedPositionsSearch.js

```diff
@@ -0,0 +1,36 @@
+import React, { useCallback, useState } from 'react';
+import { navigate } from 'gatsby';
+import SbEditable from 'storyblok-react';
+
+const EndowedPositionsSearch = (props) => {
+  const [searchTerm, setSearchTerm] = useState;
+
+  const handleInputChange = useCallback((event) => {
+    setSearchTerm(event.currentTarget.value);
+  }, []);
+  const handleSearch = useCallback(() => {
+    navigate(`/endowed-positions/search?term=${searchTerm}`);
+  }, [searchTerm]);
+  const handleKeyDown = useCallback((event) => {
+    if (event.key === 'Enter') {
+      handleSearch();
+    }
+  });
+
+  return (
+    <SbEditable content={props.blok}>
+      <input
+        // className="endowed-positions__input"
+        // id="search-input"
+        onChange={handleInputChange}
+        onKeyDown={handleKeyDown}
+        value={getSearchTerm}
```

Comment:
Also, the value here should be `searchTerm` instead of `getSearchTerm` (you also don't seem to have a `getSearchTerm` function here, though you do have a `setSearchTerm` 😄 but we aren't using either here). You already set the search term, so the value is just equal to the search term.
```suggestion
        value={searchTerm}
```